apex-music/src/player.rs

```diff
@@ -135,7 +135,7 @@ impl<T: Metadata + Sized> AsyncMetadata for T {
     type ArtistsFuture<'a> = impl Future<Output = Result<String>> + 'a
     where
         T: 'a;
-    type LengthFuture<'a> = impl Future<Output = Result<i64>> + 'a
+    type LengthFuture<'a> = impl Future<Output = Result<u64>> + 'a
```

Comment:
Is this the data type used by DBus?

Reply:
yes you can check with that code with spotify
`dbus-send --print-reply --session --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.freedesktop.DBus.Properties.GetAll string:'org.mpris.MediaPlayer2.Player'`